dist/amd/virtual-repeat.js

```diff
@@ -139,11 +139,20 @@ define(['exports', 'aurelia-dependency-injection', 'aurelia-binding', 'aurelia-t
       this.scrollContainer = this.templateStrategy.getScrollContainer(element);
       this.topBuffer = this.templateStrategy.createTopBufferElement(element);
       this.bottomBuffer = this.templateStrategy.createBottomBufferElement(element);
-      this.itemsChanged();
```

Comment:
I'm working on an implementation of horizontal scrolling, and moving this line like you did is exactly what I needed as well. So moving this to be after the checks should be good.